src/test/java/io/airlift/slice/TestFNVHash.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.slice;
+
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestFNVHash
+{
+    private byte[][] testData;
+
+    @BeforeTest
+    public void setup()
+    {
+        testData = new byte[][] {
+                "".getBytes(),
+                "foobar".getBytes(),
+                "foobar!\n".getBytes(),
+                "line 1\nline 2\nline 3".getBytes(),
+                "http://en.wikipedia.org/wiki/Fowler_Noll_Vo_hash".getBytes(),
```

Comment:
`String.getBytes()` uses the default platform encoding, so it might produce different results in different environments. Use something like `getBytes(StandardCharsets.UTF_8)`